test/test-functions

```diff
@@ -675,7 +675,7 @@ strip_binaries() {
 create_rc_local() {
     mkdir -p $initdir/etc/rc.d
     cat >$initdir/etc/rc.d/rc.local <<EOF
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I suspect the `rc-local` related fail is caused by this line. Also, most of the other tests will fail as well, since we don't install the `env` binary in the test QEMU image.

@Mic92 could you add the `env` binary to the `BASICTOOLS=` collection defined at the top of this file?

https://github.com/systemd/systemd/blob/be43dab0fbc40b9b802d1b77d6b509a5d0d1c05e/test/test-functions#L42

Reply:
Added it. Let's see how if it works. I could also skip this particular case if it runs in a controlled VM anyway.